python/gui/MaaS/templates/maas/dataset_management.html

```diff
@@ -0,0 +1,797 @@
+<!DOCTYPE html>
```

Comment:
Templates can inherit from base templates and override/fill out different elements from the parent. Using a base template here could probably cut down the size of this template down by a large portion and make it so the only things that need to be written here are the things specific to dataset management and dataset management alone.

Reply:
This and we can share common things like stylesheets.